docs/intro/features.rst

```diff
@@ -237,20 +237,65 @@ model for Russian and several other languages. Model takes as input tokenized se
 sequence of morphological labels in `UD format <http://universaldependencies.org/format.html>`__. The table below
 contains word and sentence accuracy on UD2.0 datasets.
 
-+-----------------+------------------------------------+---------------+----------------+
-|    Dataset      | Model                              | Word accuracy | Sent. accuracy |
-+-----------------+------------------------------------+---------------+----------------+
-| `UD2.0 Russian`_|`UD Pipe 1.2`_ (Straka et al., 2017)|     93.57     |     43.04      |
-+                 +------------------------------------+---------------+----------------+
-|                 |`Basic model`_                      |     95.17     |     50.58      |
-+                 +------------------------------------+---------------+----------------+
-|                 |`Pymorphy-enhanced model`_          |     96.23     |     58.00      |
-+-----------------+------------------------------------+---------------+----------------+
-
-.. _`UD2.0 Russian`: https://lindat.mff.cuni.cz/repository/xmlui/handle/11234/1-1983
+.. table::
+    :widths: auto
```

Comment:
Big table... maybe leave here only results for English and Russian, and move the rest to the morphotagger page?